src/hooks/screen/useChatScreenHook.ts

```diff
@@ -92,15 +124,22 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       }
 
       let response;
-      if (type === 'ANONYMOUS') {
-        response = await AnonymousMessageRepo.sendAnonymousMessage(selectedChannel?.id, message);
+      if (type === ANONYMOUS) {
+        response = await AnonymousMessageRepo.sendAnonymousMessage(
+          selectedChannel?.id,
+          message,
+          replyData?.id
+        );
       } else {
         response = await SignedMessageRepo.sendSignedMessage(
           selectedChannel?.id,
           message,
-          channelType
+          channelType,
+          replyData?.id
         );
       }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The sending of chat messages has been updated to handle replies and anonymous messages. Make sure that the logic correctly handles these cases.
```diff
+ if (type === ANONYMOUS) {
+   response = await AnonymousMessageRepo.sendAnonymousMessage(
+     selectedChannel?.id,
+     message,
+     replyData?.id
+   );
+  } else {
+    response = await SignedMessageRepo.sendSignedMessage(
+      selectedChannel?.id,
+      message,
+     channelType,
+     replyData?.id
+    );
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->